index.html

```diff
@@ -11,14 +11,79 @@
     />
   </head>
   <body class="app-grid">
-    <section class="header"></section>
 
-    <section class="content"></section>
+    <section class="header header__bar">
+      <p>Quiz App</p>
+    </section>
+
+    <section class="content">
+<div class="content__area">
+  
+
+    <div class="content__container container--scroll">
+      <div class="content__bookmark"></div>
+
+        <p>Question 1: </br> ipsum dolor sit amet consectetur adipisicing elit. Modi nobis nostrum 
```

Comment:
The question headline could be a `h2`-element and the question a paragraph or h3. This way we don't need the `br`-element which we usually want to avoid. Fro example:
```suggestion
        <h2>Question 1: </h2> 
          <h3>ipsum dolor sit amet consectetur adipisicing elit. Modi nobis nostrum 
```